mercury/admin.py

```diff
@@ -1,6 +1,14 @@
 from django.contrib import admin  # noqa f401
 
 # Register your models here.
-from .models import SimulatedData
+from .models import SimulatedData, TemperatureSensor, AccelerationSensor, WheelSpeedSensor, SuspensionSensor,FuelLevelSensor
```

Comment:
I'm fine with this but the line is really long and I think you'll have to update it for "black".